.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+<!-- If this pull requests solves an issue, as an example, use `Closes #31`. #31 stands for the issue number -->
+
+## Fixes Issue
+
+<!-- Remove this section if not applicable -->
+
+<!-- Example: Closes #31 -->
+
+## Changes proposed
+
+<!-- Here comes all the changes proposed through this PR -->
```

Comment:
How about ‘List all the proposed changes’ so that this is the same as the linkfree template.

Reply:
```suggestion
<!-- List all the proposed changes -->
```